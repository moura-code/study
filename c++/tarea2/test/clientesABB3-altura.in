crearClientesABB
alturaClientesABB
liberarClientesABB
crearClientesABB
crearCliente 10 Artemio Perez 21
agregarAClientesABB
alturaClientesABB
liberarClientesABB
crearClientesABB
crearCliente 5 Casimira Fernandez 52
agregarAClientesABB
crearCliente 3 Ubaldo Fernandez 19
agregarAClientesABB
alturaClientesABB
crearCliente 2 Esmeralda Lopez 21
agregarAClientesABB
alturaClientesABB
crearCliente 7 Zenobia Sanchez 33
agregarAClientesABB
alturaClientesABB
crearCliente 6 Filemon Rodriguez 22
agregarAClientesABB
alturaClientesABB
crearCliente 8 Brunilda Ruiz 31
agregarAClientesABB
alturaClientesABB
liberarClientesABB
Fin
