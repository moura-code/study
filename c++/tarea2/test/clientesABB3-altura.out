Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> El clientesABB ha sido creado de forma exitosa.
2> La altura del clientesABB es 0.
3> ClientesABB liberado con exito.
4> El clientesABB ha sido creado de forma exitosa.
5> Cliente creado de forma exitosa.
6> Se ha agregado el cliente al clientesABB de forma exitosa.
7> La altura del clientesABB es 1.
8> ClientesABB liberado con exito.
9> El clientesABB ha sido creado de forma exitosa.
10> Cliente creado de forma exitosa.
11> Se ha agregado el cliente al clientesABB de forma exitosa.
12> Cliente creado de forma exitosa.
13> Se ha agregado el cliente al clientesABB de forma exitosa.
14> La altura del clientesABB es 2.
15> Cliente creado de forma exitosa.
16> Se ha agregado el cliente al clientesABB de forma exitosa.
17> La altura del clientesABB es 3.
18> Cliente creado de forma exitosa.
19> Se ha agregado el cliente al clientesABB de forma exitosa.
20> La altura del clientesABB es 3.
21> Cliente creado de forma exitosa.
22> Se ha agregado el cliente al clientesABB de forma exitosa.
23> La altura del clientesABB es 3.
24> Cliente creado de forma exitosa.
25> Se ha agregado el cliente al clientesABB de forma exitosa.
26> La altura del clientesABB es 3.
27> ClientesABB liberado con exito.
28> Fin.
